tox.ini

```diff
@@ -32,7 +33,7 @@ allowlist_externals =
     sh
 
 [testenv:linters]
-basepython = python3.10
+basepython = python3.9
```

Comment:
`python3.10` is expected here: the typing linters are much better in python3.10 than python3.9

Reply:
Fixed